Traceback (most recent call last):
  File "/usr/share/miniconda/lib/python3.10/site-packages/jupyter_cache/executors/utils.py", line 58, in single_nb_execution
    executenb(
  File "/usr/share/miniconda/lib/python3.10/site-packages/nbclient/client.py", line 1305, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/usr/share/miniconda/lib/python3.10/site-packages/jupyter_core/utils/__init__.py", line 166, in wrapped
    return loop.run_until_complete(inner)
  File "/usr/share/miniconda/lib/python3.10/asyncio/base_events.py", line 649, in run_until_complete
    return future.result()
  File "/usr/share/miniconda/lib/python3.10/site-packages/nbclient/client.py", line 705, in async_execute
    await self.async_execute_cell(
  File "/usr/share/miniconda/lib/python3.10/site-packages/nbclient/client.py", line 1058, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/usr/share/miniconda/lib/python3.10/site-packages/nbclient/client.py", line 914, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
# Search for Kepler observations of Kepler-69
search_result = lk.search_lightcurve('Kepler-69', author='Kepler', cadence='long')
# Download all available Kepler light curves
lc_collection = search_result.download_all()
lc_collection.plot();
------------------


[0;31m---------------------------------------------------------------------------[0m
[0;31mLightkurveError[0m                           Traceback (most recent call last)
Cell [0;32mIn[2], line 4[0m
[1;32m      2[0m search_result [38;5;241m=[39m lk[38;5;241m.[39msearch_lightcurve([38;5;124m'[39m[38;5;124mKepler-69[39m[38;5;124m'[39m, author[38;5;241m=[39m[38;5;124m'[39m[38;5;124mKepler[39m[38;5;124m'[39m, cadence[38;5;241m=[39m[38;5;124m'[39m[38;5;124mlong[39m[38;5;124m'[39m)
[1;32m      3[0m [38;5;66;03m# Download all available Kepler light curves[39;00m
[0;32m----> 4[0m lc_collection [38;5;241m=[39m [43msearch_result[49m[38;5;241;43m.[39;49m[43mdownload_all[49m[43m([49m[43m)[49m
[1;32m      5[0m lc_collection[38;5;241m.[39mplot();

File [0;32m/usr/share/miniconda/lib/python3.10/site-packages/lightkurve/utils.py:563[0m, in [0;36msuppress_stdout.<locals>.wrapper[0;34m(*args, **kwargs)[0m
[1;32m    561[0m sys[38;5;241m.[39mstdout [38;5;241m=[39m devnull
[1;32m    562[0m [38;5;28;01mtry[39;00m:
[0;32m--> 563[0m     [38;5;28;01mreturn[39;00m [43mf[49m[43m([49m[38;5;241;43m*[39;49m[43margs[49m[43m,[49m[43m [49m[38;5;241;43m*[39;49m[38;5;241;43m*[39;49m[43mkwargs[49m[43m)[49m
[1;32m    564[0m [38;5;66;03m# restore to default[39;00m
[1;32m    565[0m [38;5;28;01mfinally[39;00m:
[1;32m    566[0m     sys[38;5;241m.[39mstdout [38;5;241m=[39m old_out

File [0;32m/usr/share/miniconda/lib/python3.10/site-packages/lightkurve/search.py:494[0m, in [0;36mSearchResult.download_all[0;34m(self, quality_bitmask, download_dir, cutout_size, **kwargs)[0m
[1;32m    491[0m products [38;5;241m=[39m []
[1;32m    492[0m [38;5;28;01mfor[39;00m idx [38;5;129;01min[39;00m [38;5;28mrange[39m([38;5;28mlen[39m([38;5;28mself[39m[38;5;241m.[39mtable)):
[1;32m    493[0m     products[38;5;241m.[39mappend(
[0;32m--> 494[0m         [38;5;28;43mself[39;49m[38;5;241;43m.[39;49m[43m_download_one[49m[43m([49m
[1;32m    495[0m [43m            [49m[43mtable[49m[38;5;241;43m=[39;49m[38;5;28;43mself[39;49m[38;5;241;43m.[39;49m[43mtable[49m[43m[[49m[43midx[49m[43m [49m[43m:[49m[43m [49m[43midx[49m[43m [49m[38;5;241;43m+[39;49m[43m [49m[38;5;241;43m1[39;49m[43m][49m[43m,[49m
[1;32m    496[0m [43m            [49m[43mquality_bitmask[49m[38;5;241;43m=[39;49m[43mquality_bitmask[49m[43m,[49m
[1;32m    497[0m [43m            [49m[43mdownload_dir[49m[38;5;241;43m=[39;49m[43mdownload_dir[49m[43m,[49m
[1;32m    498[0m [43m            [49m[43mcutout_size[49m[38;5;241;43m=[39;49m[43mcutout_size[49m[43m,[49m
[1;32m    499[0m [43m            [49m[38;5;241;43m*[39;49m[38;5;241;43m*[39;49m[43mkwargs[49m[43m,[49m
[1;32m    500[0m [43m        [49m[43m)[49m
[1;32m    501[0m     )
[1;32m    502[0m [38;5;28;01mif[39;00m [38;5;28misinstance[39m(products[[38;5;241m0[39m], TargetPixelFile):
[1;32m    503[0m     [38;5;28;01mreturn[39;00m TargetPixelFileCollection(products)

File [0;32m/usr/share/miniconda/lib/python3.10/site-packages/lightkurve/search.py:348[0m, in [0;36mSearchResult._download_one[0;34m(self, table, quality_bitmask, download_dir, cutout_size, **kwargs)[0m
[1;32m    344[0m download_response [38;5;241m=[39m Observations[38;5;241m.[39mdownload_products(
[1;32m    345[0m     table[:[38;5;241m1[39m], mrp_only[38;5;241m=[39m[38;5;28;01mFalse[39;00m, download_dir[38;5;241m=[39mdownload_dir
[1;32m    346[0m )[[38;5;241m0[39m]
[1;32m    347[0m [38;5;28;01mif[39;00m download_response[[38;5;124m"[39m[38;5;124mStatus[39m[38;5;124m"[39m] [38;5;241m!=[39m [38;5;124m"[39m[38;5;124mCOMPLETE[39m[38;5;124m"[39m:
[0;32m--> 348[0m     [38;5;28;01mraise[39;00m LightkurveError(
[1;32m    349[0m         [38;5;124mf[39m[38;5;124m"[39m[38;5;124mDownload of [39m[38;5;132;01m{[39;00mdownload_url[38;5;132;01m}[39;00m[38;5;124m failed. [39m[38;5;124m"[39m
[1;32m    350[0m         [38;5;124mf[39m[38;5;124m"[39m[38;5;124mMAST returns [39m[38;5;132;01m{[39;00mdownload_response[[38;5;124m'[39m[38;5;124mStatus[39m[38;5;124m'[39m][38;5;132;01m}[39;00m[38;5;124m: [39m[38;5;132;01m{[39;00mdownload_response[[38;5;124m'[39m[38;5;124mMessage[39m[38;5;124m'[39m][38;5;132;01m}[39;00m[38;5;124m"[39m
[1;32m    351[0m         )
[1;32m    352[0m path [38;5;241m=[39m download_response[[38;5;124m"[39m[38;5;124mLocal Path[39m[38;5;124m"[39m]
[1;32m    353[0m log[38;5;241m.[39mdebug([38;5;124m"[39m[38;5;124mFinished downloading.[39m[38;5;124m"[39m)

[0;31mLightkurveError[0m: Download of http://archive.stsci.edu/missions/kepler/lightcurves/0086/008692861/kplr008692861_lc_Q011111111111111111.tar failed. MAST returns ERROR: HTTPError: 404 Client Error: Not Found for url: https://mast.stsci.edu/api/v0.1/Download/file?uri=mast:KEPLER/url/missions/kepler/lightcurves/0086/008692861/kplr008692861-2009259160929_llc.fits

